### PR TITLE
Create password spec

### PR DESCRIPTION
**Pull request by tschubotz**
Spec for #65 

- [x] Use case scenarios
- [x] User interfaces

### CODE REVIEW

File: specs/create_password/01_main.rst [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+- We need the user to agree to the Terms and Privacy Policy.
+
+2. Inputs
+-----------
+
+- New password
+    - Minimum 8 characters (>=8)
+    - At least 1 number
+    - At least 1 letter
+    - No more than 2 identical characters in a row
```
Comment (sche) [inline specs/create_password/01_main.rst]: "than 3"

File: specs/create_password/01_main.rst [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+    - Needs to match
+- Fingerprint of the user
+- Terms of Use & Privacy Policy.
+
+3. Outputs
+------------
+
+- User agreed to Terms of Use and Privacy Policy.
+- Password is set.
+- Finterprint is potentially enabled.
```
Comment (sche) [inline specs/create_password/01_main.rst]: "fingerprint" typo

File: specs/create_password/01_main.rst (hunk 1 above)
Comment (tschubotz) [reply specs/create_password/01_main.rst]: why 3? xx is allowed, xxx not. That's at least what the UI says inide the app

**Review by sche — APPROVED** [review]

File: specs/create_password/01_main.rst [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+11. User proceeds.
+
+Postconditions
+++++++++++++++
+
+- User is on the home screen and can create or restore a Safe.
+- App password is set
+- User can unlock the app with the new password.
+- User cannot unlock the app with another password.
+- User can unlock the app with the fingerprint.
```
Comment (rmeissner) [inline specs/create_password/01_main.rst]: biometric information

File: specs/create_password/01_main.rst [hunk 4]
```diff
@@ -0,0 +1,167 @@
...
+
+
+Happy Case no fingerprint
```
Comment (rmeissner) [inline specs/create_password/01_main.rst]: no biometric information

File: specs/create_password/01_main.rst [hunk 5]
```diff
@@ -0,0 +1,167 @@
...
+10. User skips fingerprint setup
+
+Postconditions
+++++++++++++++
+
+- User is on the home screen and can create or restore a Safe.
+- App password is set
+- User can unlock the app with the new password.
+- User cannot unlock the app with another password.
+- User cannot unlock the app with the fingerprint.
```
Comment (rmeissner) [inline specs/create_password/01_main.rst]: biometric information

File: specs/create_password/01_main.rst [hunk 6]
```diff
@@ -0,0 +1,167 @@
...
+-----------
+
+- New password
+
+  - Minimum 8 characters (>=8)
+  - At least 1 number
+  - At least 1 letter
+  - No more than 2 identical characters in a row
+    (e.g. 111 is not allowed, 11 is fine)
+  - Maximum length?
```
Comment (rmeissner) [inline specs/create_password/01_main.rst]: any updates on this? (personally I would be against maximum length)

File: specs/create_password/01_main.rst [hunk 7]
```diff
@@ -0,0 +1,167 @@
...
+
+Problem Definition
+---------------------
+
+- We need the user to set a password for the app so:
+
+  - Nobody can open the app or confirm transactions without the password
+  - We can encrypt the private key of the device.
+
+- Most devices have a fingerprint sensor. Users should be able to use it
```
Comment (rmeissner) [inline specs/create_password/01_main.rst]: or other biometric sensors

**Review by rmeissner — APPROVED** [review]

File: specs/create_password/01_main.rst (hunk 6 above)
Comment (DmitryBespalov) [reply specs/create_password/01_main.rst]: @rmeissner why would you be against the maximum length? Don't you think that adds some security for the user inputs?

**Review by sara554-beep — APPROVED** [review]

**Pull request merged**